src/Altinn.Platform/Altinn.Platform.Events/Tests/TestingControllers/SubscriptionControllerTests .cs

```diff
@@ -0,0 +1,375 @@
+using System;
```

Comment:
just github or is there a ` ` in the file name here?
![image](https://user-images.githubusercontent.com/47737608/109313225-4412e700-7848-11eb-89e0-8cfdc4051fba.png)

Reply:
fixed